ansible/roles/test/files/ptftests/arista.py

```diff
@@ -226,7 +227,9 @@ def parse_logs(self, data):
 
         result['route_timeout'] = result_rt
 
-        if initial_time_bgp == -1 or initial_time_if == -1:
+        if self.reboot_type == 'fast-reboot' and (initial_time_bgp == -1 or initial_time_if == -1):
```

Comment:
I think we need comment here to explain what the expression mean semantically?

Reply:
put in some comments

Reply:
For WR we expect to see no flaps of any interfaces.

Reply:
@qiluo-msft mentioned that on Mellanox platforms I might see a diff of 1s for portchannel up/down. I looked at a couple of logs and saw the following behavior 
1) on Broadcom platforms, I might see the logs at times with the same timestamp for up/down or not see them at all
2) on Mellanox, same as above. At times, I see logs with a diff of 1s between up and down. 
Since we already have check_change_time() which checks the Po status, it was decided to ignore the Po logs.

Reply:
@yxieca Should we treat it as a bug?

> on Broadcom platforms, I might see the logs at times with the same timestamp for up/down